lib/phoenix/socket.ex

```diff
@@ -290,13 +280,13 @@ defmodule Phoenix.Socket do
   """
   defmacro transport(name, module, config \\ []) do
     quote do
-      @phoenix_transports Phoenix.Socket.__transport__(
+      @phoenix_transports Phoenix.Socket.merge_transport_spec(
         @phoenix_transports, unquote(name), unquote(module), unquote(config))
     end
   end
 
   @doc false
-  def __transport__(transports, name, module, config) do
+  def merge_transport_spec(transports, name, module, config) do
```

Comment:
If you remove the underscores, it will now be imported on `import Phoenix.Socket` please add them back or move the function elsewhere.